source/includes/_account.md

```diff
@@ -95,15 +95,12 @@ account.create(user, reason, comment, options)
 ```
 ```python
 accountApi = killbill.api.AccountApi()
-created_by = 'example'
+
 body = Account(name='John Doe', 
                email='john@laposte.com', 
                currency='USD')
 
-accountApi.create_account(body, 
-                          created_by, 
-                          api_key, 
-                          api_secret)
+accountApi.create_account(body,'created_by')
```

Comment:
I've also added the reason and comment like this:
```
subscriptionApi.create_subscription(body, 
                                    created_by='demo',
                                    reason='reason', 
                                    comment='comment')
```

Consider doing the same for the sake of consistency.